README.md

```diff
@@ -42,6 +43,35 @@ Prequisites:
   redis-cli ping  # Should respond with PONG
   rake db:setup
   ```
+Ubuntu:
+https://www.codingforentrepreneurs.com/blog/install-redis-mac-and-linux/ 
+- Update system
+  ```shell
+  sudo apt update
+  ```
+- Install redis server
+  ```shell
+  sudo apt install redis-server
+  ```
+- Update configuration
+  ```shell
+  sudo nano /etc/redis/redis.conf
+  ```
+- Append to configuration file:
+  ```shell
+  supervised systemd
+  ```
```

Comment:
better to avoid starting any editors, could really trigger @cbarber :trollface: 

If you need to append a file do it like this: `echo 'supervised systemd' >> /etc/redis/redis.config`

Although all of these other steps seems super complicated also. If you install docker you can have a configured running redis server running on any OS by running:

```shell
docker-compose up redis
```

and you can still run the rest of the rails instance natively

Reply:
`sudoedit ... ` is generally preferred over `sudo some-editor ...`

You'd need to wrap the command with `sudo sh -c 'echo "supervised systemd" >> /etc/redis/redis.confg'` to execute the whole command as sudo. Otherwise the append write to /etc/redis/redis.config will fail with permission denied

The docker-compose.yml doesn't expose a port for redis. There's a ticket for that:
https://www.pivotaltracker.com/story/show/171465217

Reply:
I've updated the suggested commands in the README, and removed the editor step. I tried running redis through docker, the server refused to connect. I think these separate instructions for Linux suffice until Craig's mentioned ticket is completed. @cbarber @chewnoill